app_server/test/adapters/MediaServerAdapter.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+var rfr = require('rfr');
+var Lab = require('lab');
+var lab = exports.lab = Lab.script();
+var Code = require('code');
+var Hapi = require('hapi');
+
+var MediaServerAdapter = rfr('app/adapters/MediaServerAdapter');
+var MediaServerConfig = rfr('config/ServerConfig').mediaServer;
+
+lab.experiment('getConnectionCounts', function () {
+  let mockedMediaServer = new Hapi.Server();
+  mockedMediaServer.connection({port: 8086});
+
+  mockedMediaServer.route({
+    method: 'GET',
+    path: '/connectioncounts',
+    handler: function (request, reply) {
+      reply('<?xml version="1.0"?>' +
+            '<WowzaStreamingEngine><VHost>' +
+            '</VHost></WowzaStreamingEngine>');
+    },
+  });
+
+  lab.before((done) => {
+    mockedMediaServer.start(() => done());
+  });
+
+  lab.after((done) => {
+    mockedMediaServer.stop(() => done());
```

Comment:
If we aren't really concerned with how `done()` doesn't really return anything here, why not do the same with `.stop` and do a
```lab.after((done) => mockedMediaServer.stop(() => done()));```?